pkgs/applications/emulators/emulationstation/default.nix

```diff
@@ -1,34 +1,72 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config, cmake, curl, boost, eigen
-, freeimage, freetype, libGLU, libGL, SDL2, alsa-lib, libarchive
-, fetchpatch }:
+{ lib
```

Comment:
The typical order is
- lib
- stdenv
- fetchers (asciibetical)
- all the rest (asciibetical)

No, this is not documented and no, not everyone aggrees with this, but it is the easiest way of reducing the diff sizes.